CIRG-PainTracker-Location-Body-Diagram.json

```diff
@@ -0,0 +1,1617 @@
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6767",
+            "display": "back_left_lateral_foot"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6768",
+            "display": "back_right_posterior_lateral_upper_thigh"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6769",
+            "display": "back_right_posterior_medial_upper_thigh"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6770",
+            "display": "back_right_posterior_lateral_knee"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6771",
+            "display": "back_right_posterior_medial_knee"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6772",
+            "display": "back_right_posterior_lateral_lower_leg"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6773",
+            "display": "back_right_posterior_medial_lower_leg"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6774",
+            "display": "back_right_lateral_ankle"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6775",
+            "display": "back_right_achilles_tendon"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6776",
+            "display": "back_right_medial_ankle"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6777",
+            "display": "back_right_sole_of_foot"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "6778",
+            "display": "back_right_lateral_foot"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8441",
+            "display": "front_left_anterior_neck"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8442",
+            "display": "front_left_anterior_chest"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8443",
+            "display": "front_left_anterior_upper_abdomen"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8444",
+            "display": "front_left_anterior_pelvic"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8445",
+            "display": "back_left_posterior_head"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8446",
+            "display": "back_upper_left_posterior_thorax"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8447",
+            "display": "back_upper_right_posterior_thorax"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "8448",
+            "display": "front_right_anterior_neck"
```

Comment:
@mcjustin I think this is `front_anterior_neck`.  I see no match for `front_right_anterior_neck` region on the diagram

Reply:
@achen2401  I'm pretty sure these left | right subdivisions that you've flagged here were added to dhair2 some years ago, but probably weren't in the original. 

I see them in dhair2 here:
```
grep -R front_right_anterior_chest app/ | grep -v sql | grep -v telepain
app/View/Surveys/body_diagram_paintracker.ctp:  <path class="transparent" d="M 168.97266 107.51562 C 161.15193 107.62465 153.79556 107.178 147.82031 106.45703 C 142.09511 115.76783 131.73373 129.63416 118.30273 134.91016 C 117.88513 135.07456 117.45332 135.22559 117.01172 135.36719 C 117.01172 135.36719 117.71562 150.97823 117.91602 150.36523 C 117.91602 150.36523 123.96009 174.03667 126.71289 189.38867 C 128.40129 198.80387 130.35759 206.19872 128.40039 216.76172 C 128.32719 217.15412 128.25981 217.52853 128.19141 217.89453 C 134.94381 218.14533 144.30008 217.94131 147.08008 215.16211 L 147.08008 215.16016 C 151.52162 210.71861 153.0675 174.16447 168.97266 172.52344 L 168.97266 107.51562 z" id="front_right_anterior_chest"/>
app/webroot/img/body_diagram_paintracker_report.svg:     id="front_right_anterior_chest"
```

Reply:
thanks, @mcjustin I will update the svg on my end to reflect the latest subdivisions.